specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -1966,6 +2559,30 @@
       "type": "array",
       "uniqueItems": true
     },
+    "TranscriptionPiiCategories": {
+      "description": "(Optional) describes the PII categories to return for transcription conversation detection.",
```

Comment:
Whenever there's something optional, describe the default behavior as well.

Reply:
will add

Reply:
added